### PR TITLE
fix: Add chunking in embeddings

### PR DESCRIPTION
**Pull request by Josephasafg**

### CODE REVIEW

File: libs/partners/ai21/langchain_ai21/embeddings.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
-from typing import Any, List
+from typing import Any, List, Optional
 
 from ai21.models import EmbedType
 from langchain_core.embeddings import Embeddings
 
 from langchain_ai21.ai21_base import AI21Base
 
+_DEFAULT_CHUNK_SIZE = 128
```
Comment (etang-ai21) [inline libs/partners/ai21/langchain_ai21/embeddings.py:8]: Why?

File: libs/partners/ai21/langchain_ai21/embeddings.py [hunk 2]
```diff
@@ -20,22 +22,60 @@ class AI21Embeddings(Embeddings, AI21Base):
             query_result = embeddings.embed_query("Hello embeddings world!")
     """
 
-    def embed_documents(self, texts: List[str], **kwargs: Any) -> List[List[float]]:
+    chunk_size: int = _DEFAULT_CHUNK_SIZE
+    """Maximum number of texts to embed in each batch"""
```
Comment (etang-ai21) [inline libs/partners/ai21/langchain_ai21/embeddings.py:26]: This comment should be here?

File: libs/partners/ai21/langchain_ai21/embeddings.py [hunk 3]
```diff
@@ -20,22 +22,60 @@ class AI21Embeddings(Embeddings, AI21Base):
...
+                **kwargs,
+            )
+            for chunk in chunks
+        ]
+
+        return [
+            result.embedding for response in responses for result in response.results
+        ]
+
+    def _get_len_safe_embeddings(
```
Comment (etang-ai21) [inline libs/partners/ai21/langchain_ai21/embeddings.py:73]: This should be out of the class

Comment (etang-ai21) [reply libs/partners/ai21/langchain_ai21/embeddings.py:74]: Then you'll be able to test this functionality by itself

**Review by etang-ai21 — APPROVED** [review]

File: libs/partners/ai21/langchain_ai21/embeddings.py (hunk 2 above)
Comment (Josephasafg) [reply libs/partners/ai21/langchain_ai21/embeddings.py:27]: Yes, why not?

File: libs/partners/ai21/langchain_ai21/embeddings.py (hunk 3 above)
Comment (Josephasafg) [reply libs/partners/ai21/langchain_ai21/embeddings.py]: Done

**Pull request merged**